module/VuFind/src/VuFind/ILS/Driver/SierraRest.php

```diff
@@ -276,7 +283,7 @@ class SierraRest extends AbstractBase implements
      *
      * @var int
      */
-    protected $apiVersion = 5;
+    protected $apiVersion = 5.1;
```

Comment:
Is there a reason to change the default API version as part of this PR? This may impact backward compatibility for users with older versions of Sierra (not sure how common that is at this point, though).